pyproject.toml

```diff
@@ -62,3 +62,20 @@ ignore = [
     "B905", # `zip()` without an explicit `strict=` parameter
     "B006", # Do not use mutable data structures for argument defaults"
 ]
+
+[tool.pytest.ini_options]
+addopts = ["-m default"]
+markers = [
+# only tests where backend is irrelevant:
+    "core",
```

Comment:
as in are you doing just e.g. `pytest -m core` rather than `pytest -m core tests/`?

If so would you expect that to run the tests and benchmarking? If so I can either copy the functionality to benchmarking, or even perhaps lift `conftest` to root level so that it covers both